db/seeds.rb

```diff
@@ -9,9 +9,13 @@
 require "factory_bot"
 
 if HostingEnvironment.local_development? && User.none?
+  Organisation.create!(id: 0, content_id: "af07d5a5-df63-4ddc-9383-6a666845ebe9", slug: "government-digital-service", name: "Government Digital Service")
```

Comment:
Where has this content_id come from? Is it worth maybe placing it into the `settings.yml`?
Could we also use id 1 rather than 0?

```suggestion
  Organisation.create!(id: 1, content_id: "af07d5a5-df63-4ddc-9383-6a666845ebe9", slug: "government-digital-service", name: "Government Digital Service")
```

Reply:
`content_id` comes from https://www.gov.uk/api/organisations/government-digital-service. I don't think we'd ever need to change it.

Sure we can change `id`, what's the reason for using 1 instead of 0?

Reply:
That makes sense, might be worth still sticking that value into settings.yml with a comment above it explaining where that figure comes from.

Rails tends to start id off using 1 rather than 0.